### ISSUE TITLE
App only sends media to one dropbox account when logged in and sending to multiple dropbox accounts 

### ISSUE THREAD

**Issue by roseregina** [OpenArchive/Save-app-android#191]
TLDR: it looks like you're signed into two + dropbox accounts but it puts everything on one dropbox account's folder.

While it looks like I have multiple Dropbox accounts set up in the app, I am not able to save to different Dropbox accounts. 

![TwoDropBox](https://user-images.githubusercontent.com/2231711/72030692-bb4d7c00-323e-11ea-8f91-e1b3ca6d186a.png)

Expected results: 
1. signed in with Dropbox account "Personal" and added a project "motoG" 
2. signed in with Dropbox account "Legal" and created a project "legalMotoG" in that server space (both project folders do not appear on the same Dropbox server tab)
3. Upload media to each
4. Check both Dropbox online interfaces and find the folders and images in the appropriate respective folders

Actual results: 
1. signed in with Dropbox account "Personal" and added a project "motoG" 
2. signed in with Dropbox account "Legal" and created a project "legalMotoG" in that server space (both accounts do not appear on the same Dropbox server tab)
a. when the log in page came up, I had to sign out of the "Personal" account, then signed in to the "Legal" account
3. I uploaded a couple photos to "motoG" and then uploaded a different photo to "legalMotoG" 
4. When I checked my Dropbox "Legal" account, there was an "OpenArchive SAVE" folder, but that was entirely empty
5. When I checked my Dropbox "Personal" account, the "legalMotoG" folder I had made through the view in **_Save_** for my Dropbox "Legal" account, was visible
6. When I click through to the Dropbox permission page (browser view) from the App, it shows me as being logged in to my Dropbox "Legal" account, which is not where any of the files are being saved
8. When I look at "Browse Existing Projects" for each Dropbox account, both projects appear in the _**Save**_ app.
9.  I clicked through to the Dropbox permission page (browser view) from the App, and I logged out from the account through the website. After logged out, I could still upload files from the app.

**Comment by roseregina** [comment]
As of Save 0.2.2-beta-7 on MotoG, you can successfully save to multiple Dropbox accounts and save to each [both] of them, though getting logged in to them correctly is difficult (and a separate issue #194 )
